internal/services/redis/redis_cache_resource.go

```diff
@@ -671,6 +681,13 @@ func resourceRedisCacheUpdate(d *pluginsdk.ResourceData, meta interface{}) error
 		parameters.Properties.RedisConfiguration = redisConfiguration
 	}
 
+	if v, ok := d.GetOk("access_keys_authentication_disabled"); ok {
```

Comment:
You can reference how `public_network_access_enabled` does for update. You don't need this `d.GetOK()`, instead, use `d.Get()` can get the value:

- `true` if explicitly specified to be `true`
- `false` if explicitly specified to be `false`, or absent

This is a SDKv2 behavior.